dspace-api/src/main/java/org/dspace/content/RelationshipTypeServiceImpl.java

```diff
@@ -0,0 +1,119 @@
+    protected AuthorizeService authorizeService;
+
+    public RelationshipType create(Context context) throws SQLException, AuthorizeException {
+        if (!authorizeService.isAdmin(context)) {
+            throw new AuthorizeException(
+                "Only administrators can modify relationshipType");
+        }
+        return relationshipTypeDAO.create(context, new RelationshipType());
+    }
+
+    @Override
+    public RelationshipType create(Context context, RelationshipType relationshipType)
+        throws SQLException, AuthorizeException {
+        if (!authorizeService.isAdmin(context)) {
+            throw new AuthorizeException(
+                "Only administrators can modify relationshipType");
+        }
+        return relationshipTypeDAO.create(context, relationshipType);
+    }
+
+    @Override
+    public RelationshipType findbyTypesAndLabels(Context context,EntityType leftType,EntityType rightType,
+                                                 String leftLabel,String rightLabel) throws SQLException {
+        return relationshipTypeDAO.findbyTypesAndLabels(context, leftType, rightType, leftLabel, rightLabel);
+    }
+
+    @Override
+    public List<RelationshipType> findAll(Context context) throws SQLException {
+        return relationshipTypeDAO.findAll(context, RelationshipType.class);
+    }
+
+    @Override
+    public List<RelationshipType> findByLeftOrRightLabel(Context context, String label) throws SQLException {
+        return relationshipTypeDAO.findByLeftOrRightLabel(context, label);
+    }
+
+    @Override
+    public List<RelationshipType> findByEntityType(Context context, EntityType entityType) throws SQLException {
+        return relationshipTypeDAO.findByEntityType(context, entityType);
+    }
+
+    @Override
+    public RelationshipType create(Context context, EntityType leftEntityType, EntityType rightEntityType,
+                                   String leftLabel, String rightLabel, Integer leftCardinalityMinInteger,
+                                   Integer leftCardinalityMaxInteger, Integer rightCardinalityMinInteger,
+                                   Integer rightCardinalityMaxInteger)
+        throws SQLException, AuthorizeException {
+        RelationshipType relationshipType = new RelationshipType();
+        relationshipType.setLeftType(leftEntityType);
+        relationshipType.setRightType(rightEntityType);
+        relationshipType.setLeftLabel(leftLabel);
+        relationshipType.setRightLabel(rightLabel);
+        relationshipType.setLeftMinCardinality(leftCardinalityMinInteger);
+        relationshipType.setLeftMaxCardinality(leftCardinalityMaxInteger);
+        relationshipType.setRightMinCardinality(rightCardinalityMinInteger);
+        relationshipType.setRightMaxCardinality(rightCardinalityMaxInteger);
+        return create(context, relationshipType);
+    }
+
+
+    public RelationshipType find(Context context,int id) throws SQLException {
```

Comment:
please, add the override annotation here and in the following methods

Reply:
This has been included in https://github.com/DSpace/DSpace/pull/2411